### PR TITLE
docs: link to migration guide from Blogger to Docusaurus / remove blog.johnnyreilly.com Google Analytics tag from docs

### PR DESCRIPTION
**Pull request by johnnyreilly**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

I'm really pleased that I migrated from Blogger to Docusaurus. But it wasn't straightforward. I want to make it easier for others to follow the same journey.

Also, it turns out people have been copy / pasting the example GTAG config from the docs which features my Google Analytics tag: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag#ex-config 

As a consequence, these websites are sending traffic to my Google Analytics property.  My life is now spent sending PRs like this:

- https://github.com/novuhq/novu/pull/1992
- https://github.com/undistro/undistro.io/pull/80
- https://github.com/dgiot/dgiot_doc/pull/74

I'd like to make it less likely that others make the same mistake.

If you're curious how I realised: https://twitter.com/johnny_reilly/status/1590585546451333120

## Test Plan

N/A

### Test links

Deploy preview: https://deploy-preview-8313--docusaurus-2.netlify.app/community/resources#articles

## Related issues/PRs

#8293

### CODE REVIEW

File: website/docs/api/plugins/plugin-google-gtag.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Most Docusaurus users configure this plugin through the preset options.
 // Plugin Options: @docusaurus/plugin-google-gtag
 
 const config = {
-  trackingID: 'G-226F0LR9KE',
+  trackingID: 'G-226F0LR9XX',
```
Comment (slorber) [inline website/docs/api/plugins/plugin-google-gtag.md:70]: What is this new token? Maybe it belongs to someone? 😄

Comment (johnnyreilly) [reply website/docs/api/plugins/plugin-google-gtag.md:70]: Perhaps:

`G-999F0LR9XX`

Comment (johnnyreilly) [reply website/docs/api/plugins/plugin-google-gtag.md]: I've obfuscated a bit further still

**Comment by johnnyreilly** [comment]
Thanks!

**Comment by slorber** [comment]
You may want to edit older docs versions too to remove it everywhere 😅

**Pull request merged**